app/controllers/api/groups_controller.rb

```diff
@@ -1,6 +1,12 @@
 class API::GroupsController < API::RestfulController
   load_and_authorize_resource only: :show, find_by: :key
   load_resource only: [:upload_photo, :use_gift_subscription], find_by: :key
+  skip_before_action :authenticate_user!, only: :index
```

Comment:
I was able to fix by moving `include AngularHelper` from BaseController to ApplicationController, then changing the route from `base#boot_angular_ui` to `application#boot_angular_ui`